src/extension/features/general/hide-closed-accounts/settings.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: 'HideClosedAccounts',
+  type: 'checkbox',
+  default: false,
+  section: 'general',
+  title: 'Hide Closed Accounts',
+  description:
+    'This feature hides the closed accounts section in the side menu. View the account-options popup (click your e-mail in the bottom left) to show or hide the closed accounts.',
```

Comment:
Depending on the features the user has activated, the email address may not be located in the bottom left. Suggest removing the explicit location text.

Reply:
I am fine with that change. I just coped the text from [Hide Help Button](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/src/extension/features/general/hide-help/settings.js) feature and replaced help with Closed Accounts.

Reply:
Removed the explicit `in the bottom left` from both. Mine is in the upper left as well.